01/components/Table/TableFooter.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+const TableFooter = props => {
+	const { product } = props
+	const total = product.reduce((next, number) => {
+		return next + number
+	}, 0)
+
+	return (
+		<tfoot>
+			<tr>
+				<td>Łączna wartość</td>
```

Comment:
Można też użyć atrybut `colspan` (w React to będzie `colSpan`) do łączenia komórek w całość :)